contracts/CptData.sol

```diff
@@ -21,19 +21,19 @@ pragma solidity ^0.4.4;
 import "./AuthorityIssuerData.sol";
 
 contract CptData {
-
-    uint constant private AUTHORITY_ISSUER_MAX_CPT_ID = 1999999;
-
-    uint authority_issuer_start_id = 100;
-
-    uint none_authority_issuer_start_id = 2000000;
+    // CPT ID has been categorized into 3 zones: 0 - 999 are reserved for system CPTs,
+    //  1000-2000000 for Authority Issuer's CPTs, and the rest for common WeIdentiy DIDs.
+    uint constant private AUTHORITY_ISSUER_START_ID = 1000;
+    uint constant private NONE_AUTHORITY_ISSUER_START_ID = 2000000;
+    uint private AUTHORITY_ISSUER_CURRENT_ID = 1000;
```

Comment:
AUTHORITY_ISSUER_CURRENT_ID and  NONE_AUTHORITY_ISSUER_CURRENT_ID are variables not constants. So use lower case.

Reply:
good point!